content/types/_last_connection_details.textile

```diff
@@ -0,0 +1,12 @@
+A @LastConnectionDetails@ object provides details on the last connection in a browser environment persisted when the @window beforeunload@ fired. This object is provided to the callback specified in the @recover@ attribute of "@ClientOptions@":/realtime/types#client-options. The callback in turn instructs the client library whether the connection should be recovered or not. See "connection state recovery":/realtime/connection/#connection-state-recovery for more information.
+
+h4.
+  default: Properties
+
+- recoveryKey := An opaque string obtained from the "recoveryKey":/realtime/connection/#recovery-key attribute of the "Connection object":/realtime/connection before the page was unloaded. This property is used by the library to recover the connection<br>__Type: @String@__
+
+- disconnectedAt := the time at which the previous library was abruptly disconnected before the page was unloaded. This is represented as milliseconds since epoch<br>__Type: @Integer@__
+
+- url := the URL of the previous page before the page was unloaded. A common use case for this attribute is to ensure that the previous page URL is the same as the current URL before allowing the connection to be recovered. For example, you may want the connection to be recovered only for page reloads, but not when a user navigates to a different page<br>__Type: @String@__
```

Comment:
This should refer to the fact that we use `sessionStorage`, which has origin scope, and so if the library is instanced in any page in the same origin within the same top-level browsing context then the `lastConnectionDetails` for the previous page will be given in the callback.